config.yml

```diff
@@ -195,6 +195,10 @@
 ##  Connector service log level.
 #service.log_level: INFO
 #
+##  Enable to log ids of created/indexed/deleted/updated documents during a sync.
+##    This will be logged on 'DEBUG' log level.
+#service.enable_doc_id_trace_logging: false
```

Comment:
When I first looked at the PR subject, I thought you would log in at the trace level. I think we can simplify this config to something like `enable_operation_logging`.

Another point, do you think this should be under `elasticsearch.bulk`? This config applies to the Bulk API request, and we have a similar config:
https://github.com/elastic/connectors/blob/7081964fd567f968d559e708836db7081c2f4b83/config.yml#L99-L100

Reply:
Fair point, I like the idea of putting it under `elasticsearch.bulk`. Should we use plural? `enable_operations_logging`?